Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -10,45 +10,72 @@ namespace Microsoft.MixedReality.Toolkit.Utilities.Solvers
     /// <summary>
     /// This class handles the solver components that are attached to this <see href="https://docs.unity3d.com/ScriptReference/GameObject.html">GameObject</see>
     /// </summary>
-    public class SolverHandler : ControllerFinder
+    public class SolverHandler : MonoBehaviour
     {
         [SerializeField]
         [Tooltip("Tracked object to calculate position and orientation from. If you want to manually override and use a scene object, use the TransformTarget field.")]
-        private TrackedObjectType trackedObjectToReference = TrackedObjectType.Head;
+        private TrackedObjectType trackedTargetType = TrackedObjectType.Head;
 
         /// <summary>
         /// Tracked object to calculate position and orientation from. If you want to manually override and use a scene object, use the TransformTarget field.
         /// </summary>
-        public TrackedObjectType TrackedObjectToReference
+        public TrackedObjectType TrackedTargetType
         {
-            get { return trackedObjectToReference; }
+            get { return trackedTargetType; }
             set
             {
-                if (trackedObjectToReference != value)
+                if (trackedTargetType != value)
                 {
-                    trackedObjectToReference = value;
+                    trackedTargetType = value;
                     RefreshTrackedObject();
                 }
             }
         }
 
         [SerializeField]
-        [Tooltip("When using the hand to calculate position and orientation, use this specific joint.")]
+        [Tooltip("If tracking hands or motion controllers, determines which hand(s) are valid attachments")]
```

Comment:
>If tracking hands or motion controllers, determines which hand(s) are valid attachments [](start = 18, length = 87)

I think the valid options are Left, Right, Both... right?

Because there are other options in the enum, it might be worth mentioning what we actually support (i.e. do we support "other?")

Reply:
This has come up in the past for other uses of this, and when taking an enum we either support EVERYTHING on it, or we should say what we do or do not support

---
In reply to: [308995148](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5416#discussion_r308995148) [](ancestors = 308995148)

Reply:
(y)